src/test/java/com/aws/greengrass/logmanager/model/LogFileTest.java

```diff
@@ -25,53 +28,86 @@ public class LogFileTest {
 
     @TempDir
     static Path directoryPath;
-    private final static int DEFAULT_LINES_FOR_DIGEST_NUM = 1;
+    private final static int DEFAULT_BYTES_FOR_DIGEST_NUM = 1024;
 
-    private void writeFiles(File file, int linesNeeded) throws IOException {
+    public static String getRandomTestStrings(int bytesNeeded) {
+        StringBuilder testStrings = new StringBuilder();
```

Comment:
The only bad thing about using this is it doesn't add ned lines hence doesn't simulate a real-ish scenario

Reply:
Edit I saw how you are building the strings below